src/main/java/net/greatstart/controllers/ProjectController.java

```diff
@@ -1,39 +1,88 @@
 package net.greatstart.controllers;
 
 import net.greatstart.model.Project;
+import net.greatstart.model.ProjectDescription;
+import net.greatstart.model.User;
 import net.greatstart.services.ProjectService;
+import net.greatstart.services.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.ModelAttribute;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.web.bind.annotation.ModelAttribute'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)